### ISSUE TITLE
Large deleted file still attempting to open on restart after selecting "Don't restore editors"

### ISSUE THREAD

**Issue by loopy3025** [microsoft/vscode#174247]
- VS Code Version: 1.75.1
- OS Version: Ventura 13.1

Steps to Reproduce:

1. Open VSCode with a folder
2. Create a subfolder called `/db`
3. Place an enormous sql file in the new folder
4. Change the max file size setting so it's large enough to open the enormous sql file
5. Open the enormous sql file
6. Close the editor without closing the tab
7. Delete the sql file through the OS's finder
8. Open VSCode to that folder
9. VSCode tries to open the file and errors out
10. Select "Don't restore editors" and close or restart
11. Open VSCode to that folder
12. VSCode still tries to open the file and errors out.

- I have tried this after turning off extensions and using Extension Bisect. Same result.
- I have also tried "Clear Recently Opened" to no avail

Workaround is to open VSCode to a parent folder instead.

Is there no way to delete old sessions? 

Thanks in advance.

Similar issue, but OP never responded on Extension Bisect request....
https://github.com/microsoft/vscode/issues/154175

**Comment by VSCodeTriageBot** [comment]
Thanks for reporting this issue! Unfortunately, it's hard for us to understand what issue you're seeing. Please help us out by providing a screen recording showing exactly what isn't working as expected. While we can work with most standard formats, `.gif` files are preferred as they are displayed inline on GitHub. You may find https://gifcap.dev helpful as a browser-based gif recording tool.

If the issue depends on keyboard input, you can help us by enabling screencast mode for the recording (`Developer: Toggle Screencast Mode` in the command palette). Lastly, please attach this file via the GitHub web interface as emailed responses will strip files out from the issue.

Happy coding!

**Comment by loopy3025** [comment]
OMG

**Comment by bpasero** [comment]
To clarify why I am asking for a Video, I do not understand this point of the otherwise nice steps-list:

> Select "Don't restore editors" and close or restart

**Comment by loopy3025** [comment]
Right o. Thanks.  I added that because I was thinking that, what should happen is that the window would re-open with no tabs restored. Instead, all the tabs are restored and it tries to open the same file again. This traps the user in an endless loop of never being able to open this folder again.

![Screenshot 2023-02-17 at 11 24 27 AM](https://user-images.githubusercontent.com/6729935/219709123-de499fec-424d-4569-904c-9881668015ba.png)

I figured another workaround which is to rename the folder, open the editor, close it, then rename the folder back. The "saved" session is gone.
